include/cucumber-cpp/internal/CukeDll.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CUKE_CUKEDLL_HPP_
```

Comment:
Sure, I can fix `CMakeLists.txt` to use `GenerateExportHeader`. It seems to me though that it would make `CukeDll.hpp` (or `CukeExport.h`, or however it should be named) to be obsolete. If it's ok with you, I'll just remove this file and add the necessary `include_directories` to CMake so that it finds the generated `cucumber-cpp_export.h` file.